alembic/versions/dd60c67ad3e5_insert_dictionary_data.py

```diff
@@ -0,0 +1,62 @@
+"""Insert dictionary data
+
+Revision ID: dd60c67ad3e5
+Revises: 50eb2dc8f2cd
+Create Date: 2022-08-12 13:41:50.690220
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "dd60c67ad3e5"
+down_revision = "50eb2dc8f2cd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+
+    op.execute(
+        """insert into seq_platform (`name`, `description`)
+        values ('PacBio', 'Pacific Biosciences'),
+        ('ONT', 'Oxford Nanopore Technologies'),
+        ('Illumina', 'Illumina');"""
```

Comment:
```suggestion
        ('Illumina', 'Illumina')"""
```
The SQL statements don't need a semicolon at the end. SQLAlchemy adds one anyway, so the generated SQL in the upgrades and dowgrades end up with two at the end.